### DISCUSSION TITLE
advantage of fixing scan range and masses

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#338] [Q&A]
The Github section on 'Changing default settings' reccomends to fix the scan range and mass accuracies. 

I ran an initial search of 96 biological samples (ie. 1 plate) on a timsTOF Pro with diaPASEF with the option unrelated runs ticked and averaged the reccomendations from the log. I got averages: 

scan range: 8.3
mass accuracy (MS2): 13.2ppm
MS1 accuracy : 14.4ppm. 

Using these settings (scan range 8, scan range 9, MS2 accuracy 13.2ppm & MS1 14.4ppm, MS2 fixed only, MS1 fixed only) did not change much in subsequent runs with these fixed settings. The results appeared to be equivalent.

Q1: Is there a more efficient way than taking the recommendations from the log? Is there a report table which includes them in an easier to extract format matched to the raw files ?
Q2: What is the advantage of fixing (when it does not appear to change much)?
Q3: Does the fixing improve a given particular parameter other than depth and quantification robustness? 
Q4: Does the fixing make the search faster?
Q5: When searching about 1000 raw files, is it justified  to take the average values from the first 100 raw files as representative in your experience?
Q6: Does rounding the scan range up or down make a difference in your experience?
Q7: I presume it should be advantageous to fix both MS1 and MS2 (compared to just fixing one) - correct? When setting the first value for the masses (e.g. MS1), DIA-NN automatically sets the other one to 20ppm (which can be overwritten). Does that have a certain meaning/application?

**Comment by vdemichev** [comment]
1. Yes, DIA-NN is not very sensitive to these settings. So usually sufficient to analyse just several runs, to determine the approximate averages. For example, for timsTOF which just fix both mass accuracies to 10 or 15ppm.
2. Ease of documentation & reproducibility (result does not depend on the order in which files are specified), support for reanalysis using existing .quant files. Note that 'unrelated runs' option is not suitable for quantitative analyses.
3. No.
4. Yes, for the first run in the experiment.
5. Can take average from just ~5 files.
6. Very little. 
7. More important to fix MS2. The 20ppm default is used as it's the worst accuracy one gets on any of the modern instruments with reasonable settings.

**Reply by JM-Bader** [reply]
Thanks for the swift and concise reply. 
In relation to Q2, 
- Why is it better for quantitative analyses to not use 'unrelated runs' ?
- If 'unrelated runs' is not used, the MS1 and MS2 limits are taken from the first run and then applied to all subsequent runs. That is why the order matters in that case and higher reproducibility is achieved by fixing the masses. correct?

**Reply by vdemichev** [reply]
- Variation in the scan window setting and maybe also MS2 mass tolerance would introduce extra quantification noise.
- Yes, exactly.
